modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/SystemFDSEntry.java

```diff
@@ -24,4 +26,8 @@ public default int getDefaultItemsPerPage() {
 
 	public String getRESTSchema();
 
+	public String getSymbol();
+
+	public String getTitle();
+
```

Comment:
Not completely sure, but probably the dynamic-data-list icon should be a good choice.

Reply:
I added default, showing it in the classic sample:

![image](https://github.com/user-attachments/assets/bd6e9299-80d7-495f-be72-432a037b2f2d)

Reply:
Maybe for the 'Customized Sample' we could use another icon too @markocikos?

Reply:
It is using the shopping cart icon, see image above.

Reply:
My question is regarding that. The shopping cart could not be the most appropriate icon to use in reference of a sample component. 

What about something like: `icon-order-form-pencil `for example? We don't have too much icons to represent this, but I think the customisation could be represented like as a relationship with the state of the component, not with its contents (because they could change). What do you think?